base/shell/explorer/trayprop.cpp

```diff
@@ -230,13 +230,15 @@ class CNotifySettingsPage : public CPropertyPageImpl<CNotifySettingsPage>
 private:
     HBITMAP m_hbmpTray;
     HWND m_hwndTaskbar;
+	
+	static const UINT uImageIdLookupTable[2][2][2][2];
```

Comment:
```suggestion
    static const UINT uImageIdLookupTable[2][2][2][2];
```
In all of your code, please:
- remove all trailing whitespace (see e.g. previous comment);
- use 4-space indentation instead of tabs (e.g. this suggestion).